content/code-security/code-scanning/managing-code-scanning-alerts/managing-code-scanning-alerts-for-your-repository.md

```diff
@@ -202,6 +202,10 @@ If a project has multiple alerts that you want to dismiss for the same reason, y
 
 {% ifversion remove-code-scanning-configurations %}
 
+## Re-opening dismissed alerts
+
+If you dismiss an alert but later realized that it was done by mistake, you can access the list of closed alerts. Afterwards, you can find the alert, display it and reopen it if necessary.
+
```

Comment:
Updating so that this new section isn't covered by the condition: `{% ifversion remove-code-scanning-configurations %}`